src/Holding/ViewHolding/HoldingAquisitions/useHoldingOrderLines.js

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from 'react-query';
+
+import { useOkapiKy } from '@folio/stripes/core';
+
+const useHoldingOrderLines = (id, options = {}) => {
```

Comment:
```suggestion
const useHoldingOrderLines = (id, options) => {
```
it's not really needed here

Reply:
https://github.com/folio-org/ui-inventory/pull/1397/files#diff-dd0a7f973049146dc0a0c95a89f7acad4725280dd4a5411d339eb01ed9829ba2R11

Reply:
I mean `{a: 1, b: 2, ...undefined}` will gracefully become `{a: 1, b: 2}`
So there's no need in default empty object for `options`